### PR TITLE
added directory selector

### PR DESCRIPTION
**Pull request by ziqiang2344**

### CODE REVIEW

**Review by Michelle-Lohwt — COMMENTED** [review]
@ziqiang2344 can you put up a video to show what these 2 files doing?

**Comment by ziqiang2344** [comment]
> @ziqiang2344 can you put up a video to show what these 2 files doing?

Yes sure, will do it once I complete this code. Haven't finish ammendment

**Comment by ziqiang2344** [comment]
@Michelle-Lohwt @YouQuan2007  I tried output the file path from the controller to txt and input the path as string in the Student.java. However, the output txt was not successful. Do you mind to help me check where could have been wrong? Thank you.

**Comment by Michelle-Lohwt** [comment]
Can you upload a video, cuz I cannot see what's the problem from the code

**Review by YouQuan2007 — COMMENTED** [review]
yes i agree with michelle, a video would be better in explaining the problems

**Comment by Michelle-Lohwt** [comment]
So far I haven't see a video @ziqiang2344

File: filepath.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+C:\Users\user\Downloads
```
Comment (Michelle-Lohwt) [inline filepath.txt]: What is this?

File: src/sample/classes/Student.java [hunk 2]
```diff
@@ -1,31 +1,44 @@
...
 
 
 public class Student {
+
+
     public static void main(String[] args) throws FileNotFoundException{
         List list1= new List();
         
+
+        //input the text file for the course list
```
Comment (Michelle-Lohwt) [inline src/sample/classes/Student.java]: remove unecessary comments

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
Check the comments

File: src/sample/classes/Student.java [hunk 3]
```diff
@@ -1,31 +1,6 @@
 package sample.classes;
-import java.io.FileNotFoundException;
-
-import com.itextpdf.kernel.pdf.PdfDocument;
-import com.itextpdf.kernel.pdf.PdfWriter;
-import com.itextpdf.layout.Document;
-import com.itextpdf.layout.element.List;
 
 
 public class Student {
```
Comment (Michelle-Lohwt) [inline src/sample/classes/Student.java:4]: so basically, no need this class?

File: src/sample/classes/StudentController.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+@FXML
+public void handleButtonAction(ActionEvent event) throws IOException{
+    final DirectoryChooser dirchooser = new DirectoryChooser();
+
+    Stage stage = (Stage) anchorid.getScene().getWindow();
+
+    File file = dirchooser.showDialog(stage);
+
+        //input the text file for the course list
```
Comment (Michelle-Lohwt) [inline src/sample/classes/StudentController.java:37]: the comments are still here though

File: src/sample/classes/StudentController.java [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+@FXML
+public void confirmbutton (ActionEvent event){
+    Stage stage = (Stage) anchorid.getScene().getWindow();
+    stage.close();
+
+}
+
+
+@Override
+public void initialize(URL url, ResourceBundle rb){
```
Comment (Michelle-Lohwt) [inline src/sample/classes/StudentController.java:87]: is initialize a must, if no, can remove it

File: src/sample/courseRegController.java [hunk 6]
```diff
@@ -44,7 +48,20 @@ public void LogOut(MouseEvent event) throws IOException {
...
+      Parent root1= (Parent) fxmlLoader.load();
+      Stage stage= new Stage();
+
+      stage.setTitle("Select Course List pdf directory");
+      stage.setScene(new Scene(root1));
+      stage.show();
+    } catch (Exception e){
+      System.out.println("Can't load new window");
+    }
+   //switchTo(event,"classes/Student.fxml");
```
Comment (Michelle-Lohwt) [inline src/sample/courseRegController.java]: remove unnecessary comments

**Review by Michelle-Lohwt — APPROVED** [review]

**Pull request merged**